### ISSUE TITLE
Is it possible to release 5.0.0-alpha.15, alpha 14 has a critical bug that makes virtual threads unusable.

### ISSUE THREAD

**Issue by swq8** [square/okhttp#8471]
Current alpha version has thread pinned bug #8284 #8433
need #8371 path

**Comment by anil-toshniwal** [comment]
When is the target for the release, due to the virtual thread usage and the synchronized block develop in the okhttp3, we need the fix and release.

**Comment by cpoissonnier** [comment]
Last release was more than 9 months ago. 

Is this project still alive ?

**Comment by ruiwala** [comment]
When will version 5.0.0-alpha.15 be released?

**Comment by xa-xb** [comment]
Maybe not needed in Java 24 (JEP 491: Synchronize Virtual Threads without Pinning)

**Comment by JakeWharton** [comment]
Yes, many of the changes around synchronized were reverted because of that .
